data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,268 @@ event "remnant: pantica void sprites"
+			label astralonwards
+			`	She gestures towards the exit closest to your ship, and heads off at a quick pace. Outside, she moves calmly through the bustle of the starport, stepping around quick-moving freight movers and camel wagons alike. Arriving at the <ship>, she turns, and waits for you to catch up.`
+			`	Just as you arrive and reach for the console to open the airlock, you sense a change in your surroundings. Looking around, you see that a group of Remnant gathered around the two of you, focused on Dawn. A young Remnant - scarcely more than a child - steps forward. "Thank you for your patience," he says, then steps back. An elderly Remnant steps forward. "Thank you for your curiosity," she says, then steps back. A middle-aged Remnant steps forward. "Go with our blessing, Seeker, and know that wherever you go, the Remnant go with you." She says it simply, but you feel the weight of tradition behind them. She turns to you. "And you, stranger who walks the stars... May the Embers guide your steps as you walk among us." She nods respectfully, then steps back.`
+			`	As soon as the third steps back there is a flurry of movement as each of the Remnant present quickly step forward, give her a quick hug, and then scurry off into the starport. Before you process what is going on all the Remnant gathered around you have vanished. You look at Dawn, and her expression is a mixture of awe and humility.`
+			choice
+				`	"They know you are coming back, right?"`
+				`	"What was that about?"`
+				`	"Alright, let's get going."`
+					goto astralmoving
+			`	"Well, what we are doing is dangerous. We will be crossing Korath territory, a wide expanse of uninhabited space, and then scanning a black hole. I have confidence that we will return, but nonetheless there is still a chance that we die in the process.`
+			`	"Long ago we, as a people, decided that we must face the fear of separation, the unknown, and the danger. One of the ways we do that is by holding these informal ceremonies. When one of us is going to depart on their first long distance voyage into hostile territory, a group of Remnant gather to bid them farewell. It is a way to strengthen ourselves and remind each other of our connections."`
+			choice
+				`	"Thanks for the explanation."`
+					goto astralmoving
+				`	"Kind of like a funeral, with friends and family gathering to see you off?"`
+			`	"Oh, no. I said my farewells to my friends and relatives before we left. Those who gathered to see me off today were more of a representation of Remnant society. I only recognized a few of them, but they took the time to come down to the <ship> to see me before I left. They came to see me, and wish me well, so that no matter what challenges I face, I will know that the Remnant remember me and appreicate what I am doing." Her gestures fade into contemplative stillness for a moment, then resume motion.`
```

Comment:
```suggestion
			`	"Oh, no. I said my farewells to my friends and relatives before we left. Those who gathered to see me off today were more of a representation of Remnant society. I only recognized a few of them, but they took the time to come down to the <ship> to see me before I left. They came to see me and wish me well, so that no matter what challenges I face, I will know that the Remnant remember me and appreciate what I am doing." Her gestures fade into contemplative stillness for a moment, then resume motion.`
```